clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9278,6 +9278,8 @@ def err_cast_pointer_to_non_pointer_int : Error<
 def err_nullptr_cast : Error<
   "cannot cast an object of type %select{'nullptr_t' to %1|%1 to 'nullptr_t'}0"
 >;
+def err_invalid_implicit_floating_point_cast : Error<
```

Comment:
I don't see this diagnostic emitted in any of the tests, am I just missing this?